ISISScientificComputing/autoreduce
# Add copyright statement

### Summary of work
* Created a [script](https://github.com/ISISScientificComputing/autoreduce/pull/274/files#diff-b01e75a9e195c25ab1dfa6393abd1d46) to add the copyright statement to all files in the repository
* Added the copyright statement to all files in the repository

### How to test your work
* Ensure travis passes successfully. It could be that some pylint warnings happen due to changes
* Skim through the files just to check that nothing glaring obvious has happened 


Fixes #256
